test/IRGen/ordering_x86.sil

```diff
@@ -41,4 +41,4 @@ bb0:
 // the order of features differs.
 
 // X86_64: define{{( protected)?}} swiftcc void @baz{{.*}}#0
-// X86_64: #0 = {{.*}}"target-features"="+cx16,+cx8,+fxsr,+mmx,+sahf,+sse,+sse2,+sse3,+ssse3,+x87"
+// X86_64: #0 = {{.*}}"target-features"="+avx,+crc32,+cx16,+cx8,+f16c,+fxsr,+mmx,+popcnt,+sahf,+sse,+sse2,+sse3,+sse4.1,+sse4.2,+ssse3,+x87,+xsave"
```

Comment:
We're definitely not OK with bumping the baseline requirement for swift on x86 by ~4 years without a language workgroup and/or core team discussion. I think we should really try to fix this in a different manner.

Reply:
Yeah, I think that the Language WG should be involved in that - I think that @airspeedswift was relayed this though, and @rjmccall was also CC'ed for that.

Reply:
I should also mention that the f16c removes the `__extendhfsf2` not the `__truncsfhf2` and that the latter is something that I will restore for the time being.  I think that one option might be to add `__extendhfsf2` to the support routines.